packages/vue-app/template/App.js

```diff
@@ -296,19 +297,23 @@ export default {
     <% } /* features.layouts */ %>
     <% if (isFullStatic) { %>
     getRouterBase() {
-      return (this.$router.options.base || '').replace(/\/+$/, '')
+      return withoutTrailingSlash(this.$router.options.base)
     },
     getRoutePath(route = '/') {
       const base = this.getRouterBase()
       if (base && route.startsWith(base)) {
         route = route.substr(base.length)
       }
-      return (route.replace(/\/+$/, '') || '/').split('?')[0].split('#')[0]
+      let path = route.split('?')[0].split('#')[0]
```

Comment:
Could we expose à utiliser from ufo for this @pi0 ?

Reply:
yes. see comment above 😁

Reply:
FYI it is possible with latest version of ufo to do `parsePath(route).pathname`